File: src/arte/programar/ejercicio01/java/Main.java
package arte.programar.ejercicio01.java;

import java.util.Scanner;

public class Main {

    /**
     * Escribir en consola e imprimir el mensaje
     * @param args
     */
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);

        System.out.print("Escribe un mensaje: ");
        String texto =  console.nextLine();

        System.out.println("Su mensaje es: " + texto);
    }


}

File: src/arte/programar/ejercicio01/kotlin/Main.kt
package arte.programar.ejercicio01.kotlin

import java.util.*

object Main {

    /**
     * Escribir en consola e imprimir el mensaje
     * @param args
     */
    @JvmStatic
    fun main(args: Array<String>) {
        val console = Scanner(System.`in`)

        print("Escribe un mensaje: ")
        val texto = console.nextLine()

        println("Su mensaje es: $texto")
    }
}